plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibrariesInPluginXmlTest.java

```diff
@@ -41,54 +41,124 @@
 public class AppEngineLibrariesInPluginXmlTest {
 
   private static final String APP_ENGINE_API_LIBRARY_ID = "appengine-api";
+  private static final String APP_ENGINE_ENDPOINTS_LIBRARY_ID = "appengine-endpoints";
+  private static final String OBJECTIFY_LIBRARY_ID = "objectify";
 
   @Test
-  public void testAppEngineAPIConfig() throws URISyntaxException, LibraryFactoryException {
+  public void testThereAreOnlyThreeLibraries() throws Exception {
```

Comment:
Personally I prefer the more specific throws clauses because it alerts us if a method signature changes